File: app/src/main/java/com/snaptiongame/app/data/providers/NotificationProvider.java
package com.snaptiongame.app.data.providers;

import com.snaptiongame.app.data.api.SnaptionApi;
import com.snaptiongame.app.data.providers.api.ApiProvider;

import io.reactivex.Completable;

/**
 * @author Tyler Wong
 */

public class NotificationProvider {
    private static SnaptionApi apiService = ApiProvider.getApiService();

    public static Completable refreshNotificationToken(String deviceToken) {
        return apiService.refreshNotificationToken(deviceToken);
    }
}

File: app/src/main/java/com/snaptiongame/app/data/providers/NotificationProvider.kt
@file:JvmName("NotificationProvider")

package com.snaptiongame.app.data.providers

import com.snaptiongame.app.data.providers.api.ApiProvider

import io.reactivex.Completable

/**
 * @author Tyler Wong
 */

private val apiService = ApiProvider.getApiService()

fun refreshNotificationToken(deviceToken: String): Completable = apiService.refreshNotificationToken(deviceToken)
